cas-server-core/src/main/java/org/jasig/cas/authentication/principal/CachingPrincipalAttributesRepository.java

```diff
@@ -60,6 +64,8 @@
 
     private final String cacheName = this.getClass().getSimpleName().concat(UUID.randomUUID().toString());
 
+    private transient Duration duration;
```

Comment:
I'm a bit surprised to see a `transient` attribute used for the `equals` method.